core/src/main/java/io/micronaut/core/annotation/AnnotationUtil.java

```diff
@@ -186,6 +184,16 @@ public Annotation[] getDeclaredAnnotations() {
      */
     public static final String POST_CONSTRUCT = "javax.annotation.PostConstruct";
 
+    /**
+     * The annotation attribute containing all the attributes marked as non binding.
+     */
+    public static final String NON_BINDING_ATTRIBUTE = "$nonBinding";
```

Comment:
This attribute is not included in all annotations that are using non binding annotation